lib/scrabble.rb

```diff
@@ -1,9 +1,18 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word.nil?
+    word_score(word)
```

Comment:
Like that you're using this as a sort of top-level method and breaking out some of the logic, but then handling the specifics in a supporting method.